Specs/Scene/ArcGisMapServerImageryProviderSpec.js

```diff
@@ -224,7 +224,9 @@ describe("Scene/ArcGisMapServerImageryProvider", function () {
           Resource._DefaultImplementations.createImage(
             request,
             crossOrigin,
-            deferred
+            deferred,
+            true,
+            true
```

Comment:
Ah I should have been clearer about this. The only functions that need to be updated are those that were already passing in the optional parameters like [here](https://github.com/CesiumGS/cesium/blob/d9ab862e4095ecd960555bd60a05f984c378fe23/Specs/Scene/ArcGisMapServerImageryProviderSpec.js#L337-L343). Since `createImage` now takes 6 parameters code like below needs to be changed


```
          Resource._DefaultImplementations.createImage(
            request,
            crossOrigin,
            deferred,
            true,
            true
          );
```
changes to

```
          Resource._DefaultImplementations.createImage(
            request,
            crossOrigin,
            deferred,
            true,
            false,
            true,
          );
```